components/rmrk/Gallery/Item/Sharing.vue

```diff
@@ -93,6 +101,10 @@ export default class Sharing extends Vue {
   @Prop({ default: 'Check out this cool NFT on KodaDot' }) label!: string
   @Prop({ default: () => emptyIframe }) iframe!: IFrame
   @Prop(Boolean) onlyCopyLink!: boolean
+  @Prop({ default: undefined }) currentGalleryItemImage!: {
+    image: string
+    name: string
+  }
```

Comment:
How about using a boolean value as its prop, to control the display of the download button.